python/cross-stack-resources/native-objects/native_objects/infrastructure_stack.py

```diff
@@ -0,0 +1,25 @@
+from aws_cdk import (
+    aws_lambda as lambda_,
+    core
+)
+
+class InfrastructureStack(core.Stack):
+
+    def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
+        super().__init__(scope, id, **kwargs)
+
+        my_main_func = lambda_.Function(
+            self,
+            "myMainFunction",
+            code=lambda_.InlineCode("def main(event, context)\n  print('hello, world')"),
+            handler='index.main',
+            runtime=lambda_.Runtime.PYTHON_3_7
+        )
+        
+        # We assign the function's arn to a local variable for the Object.
```

Comment:
Change this comment, it is an Object, not an ARN that is being assigned.

Reply:
Is this a self review? Should we wait for this change?